util/buildRelease/testReleaseHelp

```diff
@@ -76,6 +76,14 @@ if ($tmpstatus != 0) then
     exit($tmpstatus)
 endif
 
+$mymake protoc-gen-chpl
```

Comment:
protoc-gen-chpl will require Protobuf libraries (and, potentially, a particular version of these) in order to build. I don't think we should enable this in our regular release testing (but it would be OK to do this inside a conditional checking for some configuration / environment variable to opt in). Perhaps we can enable this block only on the system where we test protobuf today.

Reply:
At a guess, the system/config where we run those has the protobuf dependency, so let's try merging it as-is and comment it out if testing fails.